.github/workflows/create-release.yml

```diff
@@ -504,9 +527,47 @@ jobs:
             ./Windows Zip/quarto-${{needs.configure.outputs.version}}-win.zip
             ./Mac Installer/quarto-${{needs.configure.outputs.version}}-macos.pkg
             ./Mac Zip/quarto-${{needs.configure.outputs.version}}-macos.tar.gz
-            ./News/changelog.md
+            ./News/news/changelog.md
```

Comment:
There are workflows on quarto-web that use this changelog (attached to the release as `changelog.md`) to generate the release notes shown on the website. It may all be ok, but just want to raise in case this changes the path to the changelog and breaks those downstream actions.

(here https://github.com/quarto-dev/quarto-web/blob/main/.github/workflows/actions/release-info/index.js)

Reply:
Thanks for pointing that out. 

I don't think it will have some impact because we do upload from a different path, but the release will only have the filename. It seems we download directly from the release asset, so `changelog.md` will correctly be the name. 

Same way as `Windows Zip` or `Mac Zip` and not included in the name. 

Looking at my test release in fork it seems to be ok ! 

I'll what out for a Quarto Web error on this after merging this. 

Thanks again !